game/match/match.go

```diff
@@ -2006,3 +2007,18 @@ func (m *Match) handleAdminMesseges(message string, user db.User) {
 
 	}
 }
+
+func (m *Match) handleMesseges(message string) {
+
+	currentPlayer := m.CurrentPlayer().Player
```

Comment:
Won't this make it so you can win by typing /quit during the opponent's turn?

Reply:
Oversight.